GSA/usagov-benefits-eligibility
# #748 test frontend fail

## PR Summary

Fix "test frontend" fail on "unexpected token".

## Related Github Issue

- fixes #748

## Type of change

Please delete options that are not relevant.

- [X ] New feature

## Branch Name

748-test-frontend-fail

## Testing environment

Test remotely.
Test locally.

## Detailed Testing steps

Test remotely.
Pass "test / test frontend" check.

Test locally.
1. pull this branch locally
2. run npm install in terminal
3. run npm run test in terminal
4. Get result Test Suites: 30 passed, 30 total. Tests: 157 passed, 157 total.

## Checklist:
